vite.config.ts

```diff
@@ -1,14 +1,10 @@
 import { defineConfig } from 'vitest/config';
 import { sveltekit } from '@sveltejs/kit/vite';
-import { svelteTesting } from '@testing-library/svelte/vite';
 
 export default defineConfig({
-	plugins: [sveltekit(), svelteTesting()],
+	plugins: [sveltekit()],
 	test: {
 		include: ['test/**/*.{test,test.svelte}.{js,ts}'],
-		environment: 'jsdom',
-		alias: {
-			'@testing-library/svelte': '@testing-library/svelte/svelte5'
-		}
+		environment: 'jsdom'
```

Comment:
I'd suggest you already set up a coverage reporter here to be prepared for coverage testing with pipelines in the future.
Personally I most often use the v8 reporter.
These are the scripts for testing I tend to use. One for CI and one for develpment
```json
{
    // ...
    "scripts": {
        // ...
        "test:ci:unit": "vitest run --coverage",
        "test:unit": "vitest --coverage"
    }
}
```
https://vitest.dev/config/#coverage
```ts
coverage: {
	reporter: ['cobertura', 'text'],
	exclude: [
            // ...
	]
}
```

Reply:
Done! Awesome suggestion.